package.json

```diff
@@ -65,14 +64,14 @@
         "rollup-plugin-sourcemaps": "^0.4.2",
         "rollup-plugin-terser": "^5.0.0",
         "rxjs-marbles": "^5.0.0",
-        "semantic-release": "^16.0.0-beta.43",
-        "ts-jest": "^24.2.0",
+        "semantic-release": "^17.4.2",
+        "ts-jest": "^26.5.3",
         "tslint": "^5.12.1",
-        "typescript": "~3.5.2"
+        "typescript": "^4.2.3"
     },
     "peerDependencies": {
         "rxjs": ">=6.0.0-beta.0 <7",
-        "tslib": "^1.9.0"
+        "tslib": "^2.1.0"
```

Comment:
Also, are these relevant?

Reply:
_@reduxjs/toolkit_ requires an update of _typescript,_ which in turn requires an update of _tslib_ and _ts-jest,_ which in turn requires an update of _jest._

The others aren’t strictly required for the switch to _@reduxjs/toolkit;_ they fix known security issues (as reported by `npm audit`).